Documentation/RegistryDraft.md

```diff
@@ -662,16 +662,6 @@ as described by [RFC 6249].
 A client MAY use this information
 to determine its preferred strategy for downloading.
 
-A server that indexes but doesn't host packages
-SHOULD respond with a status code of `303` (See Other)
-and redirect to a hosted package archive if one is available.
-
-```http
-HTTP/1.1 303 See Other
-Content-Version: 1
-Location: https://packages.example.com/mona/LinkedList/1.1.1.zip
-```
-
```

Comment:
indexes but doesn't host packages are important to what we are trying too build, should we put this back? or is there a strong reason to remove it?

Reply:
This may have been important under the original URL-based identity scheme, but I don't think that's the case now. Scoped registry assignment and the optional `url` field in the response to `GET /{scope}/{name}` provide equivalent functionality. 

A (potentially cross-origin) redirect has the potential to create bad outcomes, such as a [redirect loop](https://en.wikipedia.org/wiki/URL_redirection#Redirect_loops) or [downgrade attack](https://en.wikipedia.org/wiki/Downgrade_attack).

If we later find this to be necessary for some desired functionality, we can always add it back.

Reply:
+@yim-lee 

It is critical that the APIs suggested in SE-0292 allows for deploying a proxy (of kind) that sits between the client and the registry serving the package content, without forcing them to make client side scoped registry assignment.

My high level thinking on how this would work was that such proxy would fully implement SE-0292 APIs but return an HTTP redirect (+relevant headers for security) to the actual registry where the package is hosted. 

@mattt @robertacroix thoughts?

Reply:
+1 on not removing this section